ui/src/components/SampleQueue/app.css

```diff
@@ -167,6 +167,11 @@
   border-bottom: 1px solid #ddd;
   padding: 0.5rem 1rem;
 }
+.queue-messages-body {
+  height: 75%;
```

Comment:
Might be cleaner to size this element with flexbox? i.e. `display: flex` and `flex-direction: column` on the parent, and then `flex: 1` on this element so it fills up the available vertical space.